src/app/components/Validator/components/ValidatorOperators.tsx

```diff
@@ -76,6 +77,11 @@ const ValidatorOperators = (props: ValidatorOperatorProps) => {
   const classes = useStyles();
   const { validator, defaultPerformance, onLoadPerformances } = props;
   const [selectedPerformancePeriod, setSelectedPerformancePeriod] = useState(defaultPerformance);
+  let operatorsPerformanceZero: number = 0;
+      validator?.operators?.forEach((operator: any) => {
+        // eslint-disable-next-line no-plusplus
+        if (operator.performance[selectedPerformancePeriod] === 0) ++operatorsPerformanceZero;
```

Comment:
Instead of adding exceptional cases to the code would be better to use `operatorsPerformanceZero += 1`